homeassistant/components/zwave_js/__init__.py

```diff
@@ -76,7 +77,20 @@ async def async_on_disconnect() -> None:
     async def async_on_initialized() -> None:
         """Handle initial full state received."""
         LOGGER.info("Connection to Zwave JS Server initialized.")
-        initialized.set()
+        # wait until all required platforms are ready
```

Comment:
Since the library creates a task that is not tracked by home assistant for the callback, we'll now not wait to fire the home assistant start event until all platforms are loaded and existing nodes discovered. Maybe that's ok?

Reply:
I suppose wel have no other solution to tackle this other that wait for a REALLY long time...